src/components/DateRangePickerInputController.jsx

```diff
@@ -58,6 +58,8 @@ const propTypes = forbidExtraProps({
 
   // i18n
   phrases: PropTypes.shape(getPhrasePropTypes(DateRangePickerInputPhrases)),
+
+  isRTL: PropTypes.bool,
```

Comment:
The Input needs the `isRTL` prop to select the correct direction of decorative arrow.